hack/validate-rbac-resourcenames.sh

```
@@ -0,0 +1,156 @@
+    local pattern="$1"
+    find "${PROJECT_ROOT}/bindata/external-secrets/resources" "${PROJECT_ROOT}/bindata/external-secrets" -name "*.yml" -exec grep -l "kind: ${pattern}" {} \; 2>/dev/null | \
+        xargs grep -h "^  name:" | \
+        awk '{print $2}' | sort -u
+}
+
+# extract CRD names from assets
+extract_crd_names() {
+    # Get CRDs from config/crd/bases
+    local crd_names=""
+    if [[ -d "${PROJECT_ROOT}/config/crd/bases" ]]; then
+        crd_names=$(find "${PROJECT_ROOT}/config/crd/bases" -name "*.yml" -o -name "*.yaml" 2>/dev/null | \
```

Shellcheck (0.10.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)